File: app/src/main/java/com/example/aac/data/fragment/list/defaults/TestListFragment.java
package com.example.aac.data.fragment.list.defaults;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.View;
import com.aac.expansion.list.AacListFragment;
import com.aac.module.pres.RequiresPresenter;
import com.chad.library.adapter.base.BaseViewHolder;

/**
 * Created by yangc on 2017/8/15.
 * E-Mail:yangchaojiang@outlook.com
 * Deprecated:
 */
@RequiresPresenter(TestFragmentPresenter.class)
public class TestListFragment extends AacListFragment<TestFragmentPresenter, String> {

    @Override
    public int getItemLayout() {
        return android.R.layout.simple_list_item_2;
    }

    /***
     * 等于0 list 大于0 GridS
     * **/
    @Override
    public int setGridSpanCount() {
        return 3;
    }

    @Override
    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        //setStartLoadMore(true);
    }

    @Override
    public void convertViewHolder(BaseViewHolder helper, String item) {
       // helper.setText(android.R.id.text1, item);
       // helper.setText(android.R.id.text2, item);
    }

    @Override
    protected boolean setOpenLazyLoad() {
        return true;
    }
}

File: kotlintest/src/main/java/com/example/kotlin/fragment/defaults/TestListFragment.kt
package com.example.kotlin.fragment.defaults

import android.os.Bundle
import android.view.View

import com.aac.expansion.list.AacListFragment
import com.aac.module.pres.RequiresPresenter
import com.chad.library.adapter.base.BaseViewHolder

/**
 * Created by yangc on 2017/8/15.
 * E-Mail:yangchaojiang@outlook.com
 * Deprecated:
 */
@RequiresPresenter(TestFragmentPresenter::class)
class TestListFragment : AacListFragment<TestFragmentPresenter, String>() {

    override fun getItemLayout(): Int {
        return android.R.layout.simple_list_item_2
    }

    override fun setGridSpanCount(): Int {
        return 3
    }

    override fun onViewCreated(view: View?, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        setStartLoadMore(true)
    }

    override fun convertViewHolder(helper: BaseViewHolder, item: String) {
        helper.setText(android.R.id.text1, item)
        helper.setText(android.R.id.text2, item)
    }

    override fun setOpenLazyLoad(): Boolean {
        return true
    }
}
